src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -16,24 +16,66 @@ public void GetAccessControl_DirectoryInfo_InvalidArguments()
         }
 
         [Fact]
+        public void GetAccessControl_DirectoryInfo_ReturnsValidObject()
+        {
+            DirectoryInfo directoryInfo = new DirectoryInfo("\\");
+
+            DirectorySecurity directorySecurity = FileSystemAclExtensions.GetAccessControl(directoryInfo);
+
+            Assert.NotNull(directorySecurity);
+        }
+
+        [Fact]
         public void GetAccessControl_DirectoryInfo_AccessControlSections_InvalidArguments()
         {
             Assert.Throws<NullReferenceException>(() => FileSystemAclExtensions.GetAccessControl((DirectoryInfo)null, new AccessControlSections()));
         }
 
         [Fact]
+        public void GetAccessControl_DirectoryInfo_AccessControlSections_ReturnsValidObject()
+        {
+            DirectoryInfo directoryInfo = new DirectoryInfo("\\");
+            AccessControlSections accessControlSections = new AccessControlSections();
+
+            DirectorySecurity directorySecurity = FileSystemAclExtensions.GetAccessControl(directoryInfo, accessControlSections);
+
+            Assert.NotNull(directorySecurity);
+        }
+
+        [Fact]
         public void GetAccessControl_FileInfo_InvalidArguments()
         {
             Assert.Throws<NullReferenceException>(() => FileSystemAclExtensions.GetAccessControl((FileInfo)null));
         }
 
         [Fact]
+        public void GetAccessControl_FileInfo_ReturnsValidObject()
+        {
+            FileInfo fileInfo = new FileInfo("\\");
```

Comment:
FWIW @the-dwyer you might find the idisposable helper classes useful eg
`            using (var temp = new TempDirectory())`
or TempFile()